templates/addons/aurora/cf.yml

```diff
@@ -68,6 +70,8 @@ Resources:
         Fn::ImportValue:
           !Sub '${App}-${Env}-VpcId'
   {{logicalIDSafe .ClusterName}}AuroraSecret:
+    Metadata:
+      'aws:copilot:description': 'A SecretsManager secret to store your DB credentials'
```

Comment:
tiny nit: when written out, "Secrets Manager" is two words (https://aws.amazon.com/secrets-manager/)
```suggestion
      'aws:copilot:description': 'A Secrets Manager secret to store your DB credentials'
```